docs/about/vopr.md

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 5
+---
+
+# Deterministic Simulation Testing
+
+Deterministic Simulation Testing (DST) is one of our favorite parts about TigerBeetle, and it is a
+key way that we improve the system's reliability.
+
+Simulation testing enables us to run the production TigerBeetle code under a wide variety of
+conditions to ensure that the cluster behaves properly. Because our simulator is deterministic based
+on a _seed_ number and the Git commit, we can perfectly reproduce any bugs discovered in testing for
+easy local debugging.
+
+## Live Simulator in the Browser
+
+You can see the simulator in action at <https://sim.tigerbeetle.com>!
+
+The three modes show TigerBeetle handling different types of network and hardware conditions -- and
+you can inject different faults yourself 🔨🧊⚡.
+
+## The VOPR
+
+_The VOPR_ is our name for our deterministic simulator. (The name was inspired by the evil AI
+supercomputer in the 1983 movie [WarGames](https://www.imdb.com/title/tt0086567/), which was called
+the War Operation Plan Response or WOPR.)
```

Comment:
Maybe instead emphasize that the supercomputer continuously simulates scenarios to learn? Been a while since I've seen the movie but idk if it was actually "evil".